client/post-editor/post-editor.jsx

```diff
@@ -569,7 +573,10 @@ const PostEditor = React.createClass( {
 
 		this.saveRawContent();
 		// TODO: REDUX - remove flux actions when whole post-editor is reduxified
-		actions.edit( { content: this.refs.editor.getContent() } );
+		const edits = merge( {}, this.props.edits, {
```

Comment:
This _should_ be fine - mutating individual elements of the state might cause subtle issues, but no one should be doing that anyway.  Done in ae1fb72.